src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -246,12 +257,8 @@ def forward(
         past_key_value: Optional[Cache] = None,
         output_attentions: bool = False,
         use_cache: bool = False,
-        **kwargs,
+        cache_position: Optional[torch.LongTensor] = None,
```

Comment:
with these changed, this is entirely `# Copied from gemma` compatible I think no? (Because all the sliding window logic is in the cache

Reply:
I have made the `forward` function copied from gemma, for the` __init__` I think there are different because gemma supports bias in projection while mistral does not